### ISSUE TITLE
preview questions

### ISSUE THREAD

**Issue by i30226** [comfyanonymous/ComfyUI#1896]
I added her to the bat file, but it didn't take effect
![image](https://github.com/comfyanonymous/ComfyUI/assets/122668340/c73f05a1-41c0-473f-8507-c72b029a2e54)

**Comment by ltdrdata** [comment]
Move `--preview-method` to end of first line.

**Comment by i30226** [comment]
![image](https://github.com/comfyanonymous/ComfyUI/assets/122668340/4cafbaa2-83b1-4cd6-9406-9ac186df327d)
I moved it to the end and it didn't take effect

**Comment by Woisek** [comment]
Yes, you moved it, but not to the first line, like he wrote. It can't have an effect if you don't place it behind a program that could be affected.

**Comment by i30226** [comment]
It has been resolved, thank you very much
